diff --git a/node_modules/@angular/build-tooling/bazel/api-gen/extraction/BUILD.bazel b/node_modules/@angular/build-tooling/bazel/api-gen/extraction/BUILD.bazel
old mode 100755
new mode 100644
index 2306077..1b28e40
--- a/node_modules/@angular/build-tooling/bazel/api-gen/extraction/BUILD.bazel
+++ b/node_modules/@angular/build-tooling/bazel/api-gen/extraction/BUILD.bazel
@@ -16,7 +16,7 @@ esbuild(
     target = "es2022",
     deps = [
         ":extract_api_to_json_lib",
-        "@npm//@angular/compiler-cli",
+        "@angular//packages/compiler-cli",
     ],
 )

@@ -26,8 +26,8 @@ ts_library(
     devmode_module = "commonjs",
     tsconfig = "@npm//@angular/build-tooling:tsconfig.json",
     deps = [
-        "@npm//@angular/compiler",
-        "@npm//@angular/compiler-cli",
+        "@angular//packages/compiler",
+        "@angular//packages/compiler-cli",
         "@npm//@bazel/runfiles",
         "@npm//@types/node",
     ],
@@ -38,8 +38,8 @@ nodejs_binary(
     name = "extract_api_to_json",
     data = [
         ":bin",
-        "@npm//@angular/compiler",
-        "@npm//@angular/compiler-cli",
+        "@angular//packages/compiler",
+        "@angular//packages/compiler-cli",
     ],
     entry_point = "bin.mjs",
     # Note: Using the linker here as we need it for ESM. The linker is not
diff --git a/node_modules/@angular/build-tooling/bazel/api-gen/rendering/BUILD.bazel b/node_modules/@angular/build-tooling/bazel/api-gen/rendering/BUILD.bazel
old mode 100755
new mode 100644
diff --git a/node_modules/@angular/build-tooling/tsconfig.json b/node_modules/@angular/build-tooling/tsconfig.json
old mode 100755
new mode 100644
index 0b54e70..048ec0e
--- a/node_modules/@angular/build-tooling/tsconfig.json
+++ b/node_modules/@angular/build-tooling/tsconfig.json
@@ -13,7 +13,12 @@
     "noImplicitReturns": true,
     "jsx": "react",
     "jsxFactory": "h",
-    "jsxFragmentFactory": "Fragment"
+    "jsxFragmentFactory": "Fragment",
+    "paths": {
+      "@angular/compiler-cli": [
+        "../../../../../../../packages/compiler-cli"
+      ]
+    }
   },
   "exclude": [
     "node_modules/**",
